### PR TITLE
tauri beta documentation

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: docs/en/usage/guides/command.md [hunk 1]
```diff
@@ -102,6 +104,11 @@ invoke('my_custom_command')
 
 ## Async Commands
 
+<Alert title="Note">
+Async commands are executed on a separate thread using the [async runtime](https://tauri.studio/en/docs/api/rust/tauri/async_runtime/fn.spawn).
+Commands without the `async` keyword are executed on the main thread, unless defined with `#[tauri::command(async)]`.
+</Alert>
```
Comment (wusyong) [inline docs/en/usage/guides/command.md:110]: This section isn't created properly.

File: docs/en/usage/contributor-guide.md [hunk 2]
```diff
@@ -2,51 +2,94 @@
...
+## Pull Request Guidelines
 
-```json
-  "tauri:source": "node ../../../cli/tauri.js/bin/tauri",
-```
+- It's OK to have multiple small commits as you work on the PR - we will let GitHub automatically squash it before merging.
+
+- If adding new feature:
+
+  - Provide convincing reason to add this feature. Ideally you should open a suggestion issue first and have it greenlighted before working on it.
```
Comment (nothingismagick) [inline docs/en/usage/contributor-guide.md]:
```suggestion
  - Provide a convincing reason to add this feature. Ideally you should open a suggestion issue first and have it greenlighted before working on it.
```

File: docs/en/usage/contributor-guide.md [hunk 3]
```diff
@@ -2,51 +2,94 @@
...
+
+- If fixing a bug:
+  - If you are resolving a special issue, add `(fix: #xxxx[,#xxx])` (#xxxx is the issue id) in your PR title for a better release log, e.g. `fix: update entities encoding/decoding (fix #3899)`.
+  - Provide detailed description of the bug in the PR, or link to an issue that does.
```
Comment (nothingismagick) [inline docs/en/usage/contributor-guide.md]:
```suggestion
  - Provide a detailed description of the bug in the PR, or link to an issue that does.
```

File: docs/en/usage/contributor-guide.md [hunk 4]
```diff
@@ -2,51 +2,94 @@
...
+
+## Development Guide
+
+**NOTE: Tauri is undergoing rapid development right now, and the docs match the latest published version of Tauri. They are horribly out of date when compared with the code in the dev branch. This contributor guide is up-to-date, but it doesn't cover all of Tauri's functions in depth. If you have any questions, don't hesitate to ask in our Discord server.**
```
Comment (nothingismagick) [inline docs/en/usage/contributor-guide.md]:
```suggestion
**NOTE: Tauri is undergoing rapid development right now, and the docs match the latest published version of Tauri. This contributor guide is up-to-date, but it doesn't cover all of Tauri's functions in depth. If you have any questions, don't hesitate to ask in our Discord server.**
```

**Review by nothingismagick — APPROVED** [review]

File: docs/en/usage/guides/command.md (hunk 1 above)
Comment (lucasfernog) [reply docs/en/usage/guides/command.md:110]: thanks!

**Pull request merged**